diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index d466ad4..589407c 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -15,13 +15,7 @@ import {
   NavFooter,
 } from '@strapi/design-system/v2';
 import { Write, Exit } from '@strapi/icons';
-import {
-  auth,
-  usePersistentState,
-  useAppInfos,
-  useTracking,
-  getFetchClient,
-} from '@strapi/helper-plugin';
+import { auth, usePersistentState, useAppInfos, useTracking, getFetchClient } from '@strapi/helper-plugin';
 import { useConfigurations } from '../../hooks';
 
 const LinkUserWrapper = styled(Box)`
@@ -40,8 +34,7 @@ const LinkUser = styled(RouterNavLink)`
   border-radius: ${({ theme }) => theme.spaces[1]};
 
   &:hover {
-    background: ${({ theme, logout }) =>
-      logout ? theme.colors.danger100 : theme.colors.primary100};
+    background: ${({ theme, logout }) => (logout ? theme.colors.danger100 : theme.colors.primary100)};
     text-decoration: none;
   }
 
@@ -82,10 +75,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
   };
 
   const handleBlur = (e) => {
-    if (
-      !e.currentTarget.contains(e.relatedTarget) &&
-      e.relatedTarget?.parentElement?.id !== 'main-nav-user-button'
-    ) {
+    if (!e.currentTarget.contains(e.relatedTarget) && e.relatedTarget?.parentElement?.id !== 'main-nav-user-button') {
       setUserLinksVisible(false);
     }
   };
@@ -110,6 +100,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
         title={menuTitle}
         icon={
           <img
+            style={{ width: '100%' }}
             src={menu.custom || menu.default}
             alt={formatMessage({
               id: 'app.components.LeftMenu.logo.alt',
@@ -169,9 +160,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
               return (
                 <NavLink
                   as={RouterNavLink}
-                  badgeContent={
-                    (link.notificationsCount > 0 && link.notificationsCount.toString()) || undefined
-                  }
+                  badgeContent={(link.notificationsCount > 0 && link.notificationsCount.toString()) || undefined}
                   to={link.to}
                   key={link.to}
                   icon={<LinkIcon />}
@@ -186,22 +175,11 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
       </NavSections>
 
       <NavFooter>
-        <NavUser
-          id="main-nav-user-button"
-          ref={buttonRef}
-          onClick={handleToggleUserLinks}
-          initials={initials}
-        >
+        <NavUser id="main-nav-user-button" ref={buttonRef} onClick={handleToggleUserLinks} initials={initials}>
           {userDisplayName}
         </NavUser>
         {userLinksVisible && (
-          <LinkUserWrapper
-            onBlur={handleBlur}
-            padding={1}
-            shadow="tableShadow"
-            background="neutral0"
-            hasRadius
-          >
+          <LinkUserWrapper onBlur={handleBlur} padding={1} shadow="tableShadow" background="neutral0" hasRadius>
             <FocusTrap onEscape={handleToggleUserLinks}>
               <Flex direction="column" alignItems="stretch" gap={0}>
                 <LinkUser tabIndex={0} onClick={handleToggleUserLinks} to="/me">
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
index a1cb3c8..6c39851 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
@@ -4,7 +4,6 @@ import { useIntl } from 'react-intl';
 import { ContentBox, useTracking } from '@strapi/helper-plugin';
 import { Flex } from '@strapi/design-system';
 import { InformationSquare, CodeSquare, PlaySquare, FeatherSquare } from '@strapi/icons';
-import CloudBox from './CloudBox';
 
 const BlockLink = styled.a`
   text-decoration: none;
@@ -20,7 +19,6 @@ const ContentBlocks = () => {
 
   return (
     <Flex direction="column" alignItems="stretch" gap={5}>
-      <CloudBox />
       <BlockLink
         href="https://strapi.io/resource-center"
         target="_blank"
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
index 1ec96b4..c4e9f97 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
@@ -41,14 +41,7 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
                     'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
                 })}
           </WordWrap>
-          {hasCreatedContentType ? (
-            <Link isExternal href="https://strapi.io/blog">
-              {formatMessage({
-                id: 'app.components.HomePage.button.blog',
-                defaultMessage: 'See more on the blog',
-              })}
-            </Link>
-          ) : (
+          {hasCreatedContentType ? null : (
             <Button size="L" onClick={onCreateCT} endIcon={<ArrowRight />}>
               {formatMessage({
                 id: 'app.components.HomePage.create',
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
index 6951fce..56395e4 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
@@ -6,106 +6,50 @@
 import React from 'react';
 import styled from 'styled-components';
 import { useIntl } from 'react-intl';
-import { useAppInfos } from '@strapi/helper-plugin';
 import { Typography, Box, Flex, Grid, GridItem } from '@strapi/design-system';
-import { Link, LinkButton } from '@strapi/design-system/v2';
-import { ExternalLink, Github, Discord, Reddit, Strapi, Twitter, Discourse } from '@strapi/icons';
-
-const StyledDiscord = styled(Discord)`
-  path {
-    fill: #7289da !important;
-  }
-`;
-
-const StyledReddit = styled(Reddit)`
-  > path:first-child {
-    fill: #ff4500;
-  }
-`;
-const StyledStrapi = styled(Strapi)`
-  > path:first-child {
-    fill: #4945ff;
-  }
-  > path:nth-child(2) {
-    fill: #fff;
-  }
-  > path:nth-child(4) {
-    fill: #9593ff;
-  }
-`;
-
-const StyledTwitter = styled(Twitter)`
-  path {
-    fill: #1da1f2 !important;
-  }
-`;
-
-const StyledDiscourse = styled(Discourse)`
-  > path:first-child {
-    fill: #231f20;
-  }
-  > path:nth-child(2) {
-    fill: #fff9ae;
-  }
-  > path:nth-child(3) {
-    fill: #00aeef;
-  }
-  > path:nth-child(4) {
-    fill: #00a94f;
-  }
-  > path:nth-child(5) {
-    fill: #f15d22;
-  }
-  > path:nth-child(6) {
-    fill: #e31b23;
-  }
-`;
+import { LinkButton } from '@strapi/design-system/v2';
+import { ExternalLink } from '@strapi/icons';
 
 const socialLinks = [
+  {
+    name: { id: 'app.components.HomePage.community.links.website', defaultMessage: 'Website' },
+    link: 'https://zapal.tech',
+    icon: <ExternalLink />,
+  },
   {
     name: { id: 'app.components.HomePage.community.links.github', defaultMessage: 'Github' },
-    link: 'https://github.com/strapi/strapi/',
-    icon: <Github fill="#7289DA" />,
-    alt: 'github',
+    link: 'https://github.com/zapal-tech',
+    icon: <ExternalLink />,
   },
   {
-    name: { id: 'app.components.HomePage.community.links.discord', defaultMessage: 'Discord' },
-    link: 'https://discord.strapi.io/',
-    icon: <StyledDiscord />,
-    alt: 'discord',
+    name: { id: 'app.components.HomePage.community.links.jira', defaultMessage: 'Jira' },
+    link: 'https://zapal.atlassian.net/jira',
+    icon: <ExternalLink />,
   },
   {
-    name: { id: 'app.components.HomePage.community.links.reddit', defaultMessage: 'Reddit' },
-    link: 'https://www.reddit.com/r/Strapi/',
-    icon: <StyledReddit />,
-    alt: 'reddit',
+    name: { id: 'app.components.HomePage.community.links.dou', defaultMessage: 'DOU' },
+    link: 'https://jobs.dou.ua/companies/zapal',
+    icon: <ExternalLink />,
   },
   {
-    name: { id: 'app.components.HomePage.community.links.twitter', defaultMessage: 'Twitter' },
-    link: 'https://twitter.com/strapijs',
-    icon: <StyledTwitter />,
-    alt: 'twitter',
+    name: { id: 'app.components.HomePage.community.links.instagram', defaultMessage: 'Instagram' },
+    link: 'https://instagram.com/zapal.tech',
+    icon: <ExternalLink />,
   },
   {
-    name: { id: 'app.components.HomePage.community.links.forum', defaultMessage: 'Forum' },
-    link: 'https://forum.strapi.io',
-    icon: <StyledDiscourse />,
-    alt: 'forum',
+    name: { id: 'app.components.HomePage.community.links.facebook', defaultMessage: 'Facebook' },
+    link: 'https://facebook.com/zapal.tech',
+    icon: <ExternalLink />,
   },
   {
-    name: { id: 'app.components.HomePage.community.links.blog', defaultMessage: 'Blog' },
-    link: 'https://strapi.io/blog?utm_source=referral&utm_medium=admin&utm_campaign=career%20page',
-    icon: <StyledStrapi />,
-    alt: 'blog',
+    name: { id: 'app.components.HomePage.community.links.twitter', defaultMessage: 'Twitter' },
+    link: 'https://twitter.com/ZapalTech',
+    icon: <ExternalLink />,
   },
   {
-    name: {
-      id: 'app.components.HomePage.community.links.career',
-      defaultMessage: 'We are hiring!',
-    },
-    link: 'https://strapi.io/careers?utm_source=referral&utm_medium=admin&utm_campaign=blog',
-    icon: <StyledStrapi />,
-    alt: 'career',
+    name: { id: 'app.components.HomePage.community.links.linkedIn', defaultMessage: 'LinkedIn' },
+    link: 'https://linkedin.com/company/zapal',
+    icon: <ExternalLink />,
   },
 ];
 
@@ -117,6 +61,11 @@ const LinkCustom = styled(LinkButton)`
   svg {
     width: ${({ theme }) => theme.spaces[6]};
     height: ${({ theme }) => theme.spaces[6]};
+    padding: 2px;
+  }
+
+  svg path {
+    fill: ${({ theme }) => theme.colors.primary500} !important;
   }
 
   span {
@@ -131,26 +80,13 @@ const GridGap = styled(Grid)`
 
 const SocialLinks = () => {
   const { formatMessage } = useIntl();
-  const { communityEdition } = useAppInfos();
 
-  const socialLinksExtended = [
-    ...socialLinks,
-    {
-      icon: <StyledStrapi />,
-      link: communityEdition
-        ? 'https://discord.strapi.io'
-        : 'https://support.strapi.io/support/home',
-      name: {
-        id: 'Settings.application.get-help',
-        defaultMessage: 'Get help',
-      },
-    },
-  ];
+  const socialLinksExtended = [...socialLinks];
 
   return (
     <Box
       as="aside"
-      aria-labelledby="join-the-community"
+      aria-labelledby="check-out-our-pages"
       background="neutral0"
       hasRadius
       paddingRight={5}
@@ -162,33 +98,28 @@ const SocialLinks = () => {
       <Box paddingBottom={7}>
         <Flex direction="column" alignItems="stretch" gap={5}>
           <Flex direction="column" alignItems="stretch" gap={3}>
-            <Typography variant="delta" as="h2" id="join-the-community">
+            <Typography variant="delta" as="h2" id="check-out-our-pages">
               {formatMessage({
                 id: 'app.components.HomePage.community',
                 defaultMessage: 'Join the community',
               })}
             </Typography>
-            <Typography textColor="neutral600">
-              {formatMessage({
-                id: 'app.components.HomePage.community.content',
-                defaultMessage:
-                  'Discuss with team members, contributors and developers on different channels',
-              })}
-            </Typography>
           </Flex>
-          <Link href="https://feedback.strapi.io/" isExternal endIcon={<ExternalLink />}>
-            {formatMessage({
-              id: 'app.components.HomePage.roadmap',
-              defaultMessage: 'See our road map',
-            })}
-          </Link>
         </Flex>
       </Box>
       <GridGap>
         {socialLinksExtended.map(({ icon, link, name }) => {
           return (
             <GridItem col={6} s={12} key={name.id}>
-              <LinkCustom size="L" startIcon={icon} variant="tertiary" href={link} isExternal>
+              <LinkCustom
+                size="L"
+                startIcon={icon}
+                variant="tertiary"
+                href={link}
+                isExternal
+                target="_blank"
+                rel="noopener noreferrer"
+              >
                 {formatMessage(name)}
               </LinkCustom>
             </GridItem>
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index 72836a4..25c2f43 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -5,13 +5,11 @@
 
 import React, { memo, useMemo } from 'react';
 import { FormattedMessage } from 'react-intl';
-import styled from 'styled-components';
 import { Helmet } from 'react-helmet';
 import { useHistory } from 'react-router-dom';
 import { LoadingIndicatorPage, useGuidedTour } from '@strapi/helper-plugin';
 import { Layout, Main, Box, Grid, GridItem } from '@strapi/design-system';
 import useLicenseLimitNotification from 'ee_else_ce/hooks/useLicenseLimitNotification';
-import cornerOrnamentPath from './assets/corner-ornament.svg';
 import { useModels } from '../../hooks';
 import isGuidedTourCompleted from '../../components/GuidedTour/utils/isGuidedTourCompleted';
 import GuidedTourHomepage from '../../components/GuidedTour/Homepage';
@@ -19,24 +17,13 @@ import SocialLinks from './SocialLinks';
 import HomeHeader from './HomeHeader';
 import ContentBlocks from './ContentBlocks';
 
-const LogoContainer = styled(Box)`
-  position: absolute;
-  top: 0;
-  right: 0;
-
-  img {
-    width: ${150 / 16}rem;
-  }
-`;
-
 const HomePage = () => {
   // Temporary until we develop the menu API
   const { collectionTypes, singleTypes, isLoading: isLoadingForModels } = useModels();
   const { guidedTourState, isGuidedTourVisible, isSkipped } = useGuidedTour();
   useLicenseLimitNotification();
 
-  const showGuidedTour =
-    !isGuidedTourCompleted(guidedTourState) && isGuidedTourVisible && !isSkipped;
+  const showGuidedTour = !isGuidedTourCompleted(guidedTourState) && isGuidedTourVisible && !isSkipped;
 
   const { push } = useHistory();
   const handleClick = (e) => {
@@ -48,9 +35,7 @@ const HomePage = () => {
   const hasAlreadyCreatedContentTypes = useMemo(() => {
     const filterContentTypes = (contentTypes) => contentTypes.filter((c) => c.isDisplayed);
 
-    return (
-      filterContentTypes(collectionTypes).length > 1 || filterContentTypes(singleTypes).length > 0
-    );
+    return filterContentTypes(collectionTypes).length > 1 || filterContentTypes(singleTypes).length > 0;
   }, [collectionTypes, singleTypes]);
 
   if (isLoadingForModels) {
@@ -63,16 +48,10 @@ const HomePage = () => {
         {(title) => <Helmet title={title[0]} />}
       </FormattedMessage>
       <Main>
-        <LogoContainer>
-          <img alt="" aria-hidden src={cornerOrnamentPath} />
-        </LogoContainer>
         <Box padding={10}>
           <Grid>
             <GridItem col={8} s={12}>
-              <HomeHeader
-                onCreateCT={handleClick}
-                hasCreatedContentType={hasAlreadyCreatedContentTypes}
-              />
+              <HomeHeader onCreateCT={handleClick} hasCreatedContentType={hasAlreadyCreatedContentTypes} />
             </GridItem>
           </Grid>
           <Grid gap={6}>
diff --git a/node_modules/@strapi/admin/index.html b/node_modules/@strapi/admin/index.html
index a00a161..d6ba7ae 100644
--- a/node_modules/@strapi/admin/index.html
+++ b/node_modules/@strapi/admin/index.html
@@ -6,11 +6,12 @@
     <!-- Make the page mobile compatible -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="mobile-web-app-capable" content="yes" />
-    <title>Strapi Admin</title>
+    <meta name="robots" content="noindex" />
+    <title>Zapal | Admin Panel</title>
   </head>
   <body>
     <!-- The app hooks into this div -->
-    <div style="min-height:100%" id="app"></div>
+    <div style="min-height: 100%" id="app"></div>
     <!-- A lot of magic happens in this file. HtmlWebpackPlugin automatically includes all assets (e.g. bundle.js, main.css) with the correct HTML tags, which is why they are missing in this HTML file. Don't add any assets here! (Check out webpackconfig.js if you want to know more) -->
   </body>
 </html>
